cli/command_repository_create.go

```diff
@@ -101,7 +101,7 @@ func (c *commandRepositoryCreate) ensureEmpty(ctx context.Context, s blob.Storag
 func (c *commandRepositoryCreate) runCreateCommandWithStorage(ctx context.Context, st blob.Storage) error {
 	err := c.ensureEmpty(ctx, st)
 	if err != nil {
-		return errors.Wrap(err, "unable to get repository storage")
+		//return errors.Wrap(err, "unable to get repository storage")
```

Comment:
that's probably not intentional, revert?

Reply:
This is the problem I mentioned above. If user wants to have this .storageconfig in the bucket before the repository is created, so the even the initial blobs are under the storage class treatment, this error here will prevent repository from being created, because it's not empty. Of course, we could tell users "only create .storageconfig AFTER repository is created", but I feel it unnecessarily complicate our story.

Is there a way to just ignore the .storageconfig when checking empty?

Reply:
you can change `ListBlobs` and not invoke the callback for this one file. This is probably the simplest way to hide provider-internal files.

Reply:
Neat! Done.